RecoLocalCalo/HcalRecAlgos/interface/HcalDeterministicFit.h

```diff
@@ -167,11 +163,10 @@ void HcalDeterministicFit::apply(const CaloSamples & cs, const std::vector<int>
   if (ch5<1) {
     ch5=0;
   }
```

Comment:
Lines 163-165 are irrelevant since ch5 is not referenced again. They should be deleted. Same for lines 157-159, since ch3 is not referenced again either:
```
 if (ch3<1) {
    ch3=0;
  }
```